tests/cephfs/cephfs_mirroring/test_cephfs_mirror_disconnect.py

```diff
@@ -0,0 +1,461 @@
+            sudo=True,
+            src="tests/cephfs/cephfs_mirror_upgrade/md5sum_script.sh",
+            dst=f"/root/{md5sum}",
+        )
+        snap1 = "snap_k1_nw"
+        snap2 = "snap_f1_nw"
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir -p {kernel_mounting_dir_1}/{subvol1_path}/{mounting_dir};"
+            f"python3 /home/cephuser/smallfile/smallfile_cli.py "
+            f"--operation create --threads 1 --file-size 1 "
+            f"--files 5 --files-per-dir 1 --dirs-per-dir 2 --top "
+            f"{kernel_mounting_dir_1}/{subvol1_path}/{mounting_dir}",
+        )
+
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"mkdir {kernel_mounting_dir_1}{subvol1_path}.snap/{snap1}",
+        )
+        fsid = fs_mirroring_utils.get_fsid(cephfs_mirror_node[0])
+        log.info(f"fsid on ceph cluster : {fsid}")
+        daemon_names = fs_mirroring_utils.get_daemon_name(source_clients[0])
+
+        asok_files = fs_mirroring_utils.get_asok_file(
+            cephfs_mirror_node, fsid, daemon_names
+        )
+        log.info(f"Admin Socket file of cephfs-mirror daemon : {asok_files}")
+        filesystem_id = fs_mirroring_utils.get_filesystem_id_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"filesystem id of {source_fs} is : {filesystem_id}")
+        peer_uuid = fs_mirroring_utils.get_peer_uuid_by_name(
+            source_clients[0], source_fs
+        )
+        log.info(f"peer uuid of {source_fs} is : {peer_uuid}")
+
+        cmd = (
+            f"cd /var/run/ceph/{fsid}/ ; ceph --admin-daemon {asok_files[0]} fs mirror peer status "
+            f"{source_fs}@{filesystem_id} {peer_uuid} -f json"
+        )
+        for node in cephfs_mirror_node:
+            node.exec_command(sudo=True, cmd="yum install -y ceph-common --nogpgcheck")
+        retry_exec_command = retry(CommandFailed, tries=3, delay=30)(
+            cephfs_mirror_node[0].exec_command
+        )
+        while True:
+            out, _ = retry_exec_command(sudo=True, cmd=cmd)
+            data = json.loads(out)
+            log.info(data)
+            for path, status in data.items():
+                state = status.get("state")
+                if state == "syncing":
+                    fs_util_ceph1.network_disconnect_v1(
+                        cephfs_mirror_node[0], sleep_time="90"
+                    )
+                    break
+                time.sleep(3)
+                last_synced_snap = status.get("last_synced_snap")
+                if last_synced_snap:
+                    if last_synced_snap.get("name") == snap1:
+                        break
+            if state == "syncing" or (
+                last_synced_snap and last_synced_snap.get("name") == snap1
+            ):
+                break
+        log.info("=" * 50)
+        log.info("Sceneario 1 : Disconnect the mirroring Node network for 90 seconds.")
```

Comment:
This log.info message should be before network disconnect op starts, at line 223?